token_server/README.md

```diff
@@ -4,104 +4,109 @@ This is an optional component for the mapping page, that allows any user to view
+    for GiveDirectly). You must be logged in as the main owner to access it.
 
 ### Set up new Google Cloud instance
 
-*  You should not normally need to do this unless you had to switch Google
-   accounts, as described in [INITIAL_SETUP.md](../docs/INITIAL_SETUP.md).
-*  If you have already followed the instructions at
-   [INITIAL_SETUP.md](../docs/INITIAL_SETUP.md) to set up the mapping page in
-   the first place, most of the work needed here is already done. The only
-   additional work is to create a service account that is whitelisted for
-   EarthEngine access (currently `earthengine-token-provider`) and give it the
-   "Service Account Token Creator" role in the
-   [IAM page](https://console.developers.google.com/iam-admin/iam?project=mapping-crisis).
-
-*  If you are serving tokens from Google App Engine, and not from Amazon, then
-   you should also give the default AppEngine service account (currently
-   `mapping-crisis@appspot.gserviceaccount.com`) the "Service Account Token
-   Creator" role, so that it can create tokens for the service account you
-   created above.
+*   You should not normally need to do this unless you had to switch Google
+    accounts, as described in [INITIAL_SETUP.md](../docs/INITIAL_SETUP.md).
+*   If you have already followed the instructions at
+    [INITIAL_SETUP.md](../docs/INITIAL_SETUP.md) to set up the mapping page in
+    the first place, most of the work needed here is already done. The only
+    additional work is to create a service account that is whitelisted for
+    EarthEngine access (currently `earthengine-token-provider`) and give it the
+    "Service Account Token Creator" role in the
+    IAM page ([here](https://console.developers.google.com/iam-admin/iam?project=mapping-crisis)
+    for GiveDirectly).
+
+*   If you are serving tokens from Google App Engine, and not from Amazon, then
+    you should also give the default AppEngine service account (for
+    GiveDirectly, `mapping-crisis@appspot.gserviceaccount.com`) the "Service
+    Account Token Creator" role, so that it can create tokens for the service
+    account you created above.
 
 ## Run server on Amazon Elastic Beanstalk
 
-*  Deploy using
-   [these instructions](https://aws.amazon.com/getting-started/tutorials/deploy-app-command-line-elastic-beanstalk/).
-   Since the project already exists on Amazon Elastic Beanstalk, you just need
-   to run `eb deploy` after running `eb init` in order to log in.
+*   Deploy using
+    [these instructions](https://aws.amazon.com/getting-started/tutorials/deploy-app-command-line-elastic-beanstalk/).
+    Since the project already exists on Amazon Elastic Beanstalk, you just need
+    to run `eb deploy` after running `eb init` in order to log in.
 
-*  The server URL will be entered here once we have GiveDirectly's account.
+*   The server URL will be entered here once we have GiveDirectly's account.
```

Comment:
maybe put this in parentheses or something so it's clear that it's not an instruction for users? To be honest, my guess is we'll never actually use this section.

Reply:
Updated to I think what you meant